homeassistant/components/nextcloud/entity.py

```diff
@@ -12,19 +14,24 @@ class NextcloudEntity(CoordinatorEntity[NextcloudDataUpdateCoordinator]):
     """Base Nextcloud entity."""
 
     _attr_has_entity_name = True
-    _attr_icon = "mdi:cloud"
 
     def __init__(
-        self, coordinator: NextcloudDataUpdateCoordinator, item: str, entry: ConfigEntry
+        self,
+        coordinator: NextcloudDataUpdateCoordinator,
+        item: str,
+        entry: ConfigEntry,
+        desc: EntityDescription,
     ) -> None:
         """Initialize the Nextcloud sensor."""
         super().__init__(coordinator)
         self.item = item
-        self._attr_translation_key = slugify(item)
         self._attr_unique_id = f"{coordinator.url}#{item}"
         self._attr_device_info = DeviceInfo(
-            name="Nextcloud",
+            configuration_url=coordinator.url + "/settings/admin/serverinfo",
+            entry_type=DeviceEntryType.SERVICE,
```

Comment:
Can we do the device changes in a different PR?

Reply:
@escoand do you want to move this to a new PR?

Reply:
unrelated changes were removed with 2926a88